v63002/u_inref/gtm8766.csh

```diff
@@ -22,9 +22,17 @@ $GDE change -segment DEFAULT -access_method=BG >& accessmethod.out
 if ($status) then
 	echo "# UNABLE TO CHANGE ACCESS METHOD"
 endif
-echo "# In the current version, 2MB is the maximum value accepted by both MUPIP and GDE. In previous versions"
```

Comment:
I would like this line reinserted but with a modification indicating 2Mb is for 64-bit builds and 64Kb is for 32-bit builds.